### ISSUE TITLE
Add support for gofiber

### ISSUE THREAD

**Issue by ajipandean-w** [tech-djoin/go-prometheus#1]

**Comment by Alitindrawan24** [comment]
Nice suggest my bro, and maybe it will be possible to make support for various go web frameworks?

**Comment by ajipandean-w** [comment]
Yes, but maybe we can go from conducting a mini-research to know which widely-used go web frameworks in the community. Since it will takes much effort supporting unpopular frameworks, except they beg for it or makes contribution here, lmao.

**Comment by Alitindrawan24** [comment]
> Yes, but maybe we can go from conducting a mini-research to know which widely-used go web frameworks in the community. Since it will takes much effort supporting unpopular frameworks, except they beg for it or makes contribution here, lmao.

Yes nice, by the way in release of [v0.3.0](https://github.com/tech-djoin/go-prometheus/releases/tag/v0.3.0), this package already support multiple framework/http library. You can see on wrapper folder.

Note: Readme.md not updated yet hehehe

**Comment by ajipandean-w** [comment]
Uuu, nice. I'll go check it later.

**Comment by Alitindrawan24** [comment]
Holla Good news @ajipandean-w after release of [v0.4.2](https://github.com/tech-djoin/go-prometheus/releases/tag/v0.4.2), this package already support Fiber Framework smoothly. Can you try out on your project now, thanku 😊
